### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by doanguyen**

### CODE REVIEW

File: src/popup/App.vue [hunk 1]
```diff
@@ -127,18 +127,30 @@
...
-          </div>
+          <table class="table table-sm table-borderless">
+            <colgroup>
+              <col span="1" style="width: 80%;" />
+              <col span="1" style="width: 20%;" />
+            </colgroup>
+            <tbody>
+              <tr v-for="alias in existing" v-bind:key="alias">
+                <td>
+                  <a v-b-tooltip.hover v-bind:title="alias" class="small-text">{{ alias | truncate(50, "...") }}</a>
```
Comment (nguyenkims) [inline src/popup/App.vue]: The UI is much nicer, 👍. Do you know if we can enable the "copy on click", meaning when user clicks on the alias, the alias will be copied to the clipboard. This way user doesn't have to click on the "copy" button which is quite small.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,9 @@
 SimpleLogin Chrome/Firefox extension
 ---
-<p>
```
Comment (nguyenkims) [inline README.md]: ah this allows to having some nice badges on the readme, why do we need to remove them?

File: README.md [hunk 3]
```diff
@@ -1,23 +1,9 @@
...
-
-<a href="./LICENSE">
-<img src="https://img.shields.io/github/license/simple-login/app">
-</a>
-
-</p>
 
 SimpleLogin is the **open-source** privacy-first email alias and Single Sign-On (SSO) Identity Provider.
 
-More info on our website at https://simplelogin.io
```
Comment (nguyenkims) [inline README.md]: The phrase seems more natural with "More info ..." than just the link :)

File: README.md (hunk 2 above)
Comment (doanguyen) [reply README.md]: This is due to my stupid git command, fixed in the last push

**Pull request merged**